### PR TITLE
relocating location for the census flag to the top of the page below …

### PR DESCRIPTION
**Pull request by yomiodeleye**
…page title

### What

Relocating location for the census flag to the top of the page below page title

### How to review

Screenshot 2022-10-04 at 5 57 55 am

### Who can review

FD

### CODE REVIEW

File: assets/templates/partials/release/status-header.tmpl [hunk 1]
```diff
@@ -1,4 +1,27 @@
 <div>
+  <div class="ons-u-pb-s">
+    {{ if .Description.Census2021 }}
+      <div
+        {{ if or (eq .PublicationState.Type "upcoming") (eq .PublicationState.Type "cancelled") }}
+          class="ons-u-pb-m"
+        {{ end }}
```
Comment (rafahop) [inline assets/templates/partials/release/status-header.tmpl:7]: Is this still required? My understanding is that this class is needed to add a gap with the "this release is not yet published" div but with this change that one will not be the next element.

File: assets/templates/partials/release/status-header.tmpl [hunk 2]
```diff
@@ -1,4 +1,27 @@
 <div>
+  <div class="ons-u-pb-s">
+    {{ if .Description.Census2021 }}
```
Comment (rafahop) [inline assets/templates/partials/release/status-header.tmpl:3]: Won't this create an empty div when not a census release?

File: assets/templates/partials/release/status-header.tmpl (hunk 1 above)
Comment (yomiodeleye) [reply assets/templates/partials/release/status-header.tmpl]: yes you are correct. I have fix this.

File: assets/templates/partials/release/status-header.tmpl (hunk 2 above)
Comment (yomiodeleye) [reply assets/templates/partials/release/status-header.tmpl]: yes that true and i have fix this.

**Review by rafahop — APPROVED** [review]
Looks good to me!

**Pull request merged**